### ISSUE TITLE
Re-import/update from Lutris?

### ISSUE THREAD

**Issue by Freso** [sonic2kk/steamtinkerlaunch-tweaks#1]
I was looking through some of the tweaks and [`9200.conf`](https://github.com/frostworx/steamtinkerlaunch-tweaks/blob/6f247c127810011a44f99658ac22333bc7462def/tweaks/9200.conf) has a number of tweaks seemingly imported from Lutris, but none of the [installers in Lutris](https://lutris.net/games/rage/) contain anything about any winetricks packages, so I guess this might be from an older installer.

I don’t see a script to reimport/update tweaks in this repository, but is this something that could be done? Or are these tweaks essentially discontinued/deprecated?

**Comment by frostworx** [comment]
I abandoned the tweaks long ago before I passed over steamtinkerlaunch development to sonic2kk and only left some basix tweak
functionality in steamtinkerlaunch itself.
Since then you're the only one who looked into it and I do not even have a clue if any tweak integration is left in steamtinkerlaunch.

TL;DR:

this repo is dead, possibly still functional, but in any case, VERY outdated :)

**Comment by frostworx** [comment]
np
